test_scripts/API/LastMileNavigation/GetWayPoints/010_GetWayPoints_without_wayPoints.lua

```diff
@@ -0,0 +1,52 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_requirements/issues/25
+-- Use case: https://github.com/smartdevicelink/sdl_requirements/blob/master/detailed_docs/embedded_navi/Get%20Destination_and_Waypoints.md
+-- Item: Use Case 1: Main Flow
+--
+-- Requirement summary:
+-- [GetWayPoints] As a mobile app I want to send a request to get the details of the destination 
+-- and waypoints set on the system so that I can get last mile connectivity.
+--
+-- Description:
+-- In case:
+-- 1) mobile application sends valid and allowed by Policies GetWayPoints_request to SDL
+-- SDL must:
+-- 1) Transfer GetWayPoints_request to HMI
+-- 2) Respond with <resultCode> received from HMI to mobile application
+
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonLastMileNavigation = require('test_scripts/API/LastMileNavigation/commonLastMileNavigation')
+
+local response = {}
+
+--[[ Local Functions ]]
+local function GetWayPoints(self)
+  local params = { 
+    wayPointType = "ALL"
+  }
+  local cid = self.mobileSession1:SendRPC("GetWayPoints", params)
+
+  response.appID = commonLastMileNavigation.getHMIAppId()
+  EXPECT_HMICALL("Navigation.GetWayPoints", params):ValidIf(function(_, data)
+    return data.params.appID == commonLastMileNavigation.getHMIAppId()
+  end)
+  :Do(function(_,data)
+    self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", response)
+  end)
+  self.mobileSession1:ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
```

Comment:
@LitvinenkoIra If purpose of the script is to check empty response then appropriate validation has to be implemented for checking response SDL->App